sdk/src/builder.rs

```diff
@@ -1790,8 +1791,9 @@ impl Builder {
         let mut store = Store::new();
         store.commit_claim(claim)?;
 
-        let signer = Self::working_store_signer()?;
-        store.get_box_hashed_embeddable_manifest(signer.as_ref(), settings)
+        //store.to_jumbf_internal(1000)
+        store.get_data_hashed_manifest_placeholder(100, "application/c2pa")
```

Comment:
How did you find/pick the 100? Educated guess?

Reply:
It doesn't matter since there's no signature we are adding. It would probably work with 0 but sometimes you run into edge cases, so I just picked a small but non-zero size. I looked into creating a well structured but fake signature, but that was too complicated.